internal/provider/resource_directory_role_collection.go

```diff
@@ -118,6 +120,9 @@ func (rs *directoryRoleCollectionType) Read(ctx context.Context, req resource.Re
 	state.Name = types.StringValue(cliRes.Name)
 	state.Description = types.StringValue(cliRes.Description)
 
+	// Setting ID of state - required by hashicorps terraform plugin testing framework for Import . See issue https://github.com/hashicorp/terraform-plugin-testing/issues/84
+	state.Id = state.Name
```

Comment:
Should follow the id format used by import: `<directoryid>,<role-collection-name>`

Reply:
Adjusted